src/app/sol-warnings.js

```diff
@@ -0,0 +1,34 @@
+// source: just a big string of the current tab's source code.
+//
+// compiledResult: an object with different keys based on success/failure of compilation.
+//   success keys: 'contracts' and 'sources' (contains AST)
+//   error key: 'error' (contains compile error string)
+
+function find (source, compiledResult) {
+  var messages = [];
+
+  if (source) {
+    if (source.match(/\.\s*send\s*\(/)) {
+      messages.push('Caution: You are using the send method.\nPlease read <a target="_blank" href="http://hackingdistributed.com/2016/06/16/scanning-live-ethereum-contracts-for-bugs/">this</a> before deploying your code.');
```

Comment:
General warning: Yes - if we add such a warning, it should be always present. We cannot detect any external function call and we certainly do not want this warning for every single external function call.